tool/template-calculator.html

```diff
@@ -25,7 +25,7 @@
     <section class="row mb-49">
       <h6 class="section-marker">{% trans %}Sustainability{% endtrans %}</h6>
       <div class="col-sm-4 d-sm-none d-md-block">
-        <img class="mw-100" src="./img/green/green-graphic.png">
+        <img class="mw-100" src="./img/green/green-graphic.png" alt="{% trans %}Sustainability icon{% endtrans %}">
```

Comment:
I think for cases like this it's better to have an actually empty alt text since the image is purely decorative. (Read the link from my previous comment for details)